Automattic/jetpack
# VideoPress: connect videos Filter UI with the data handling

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: connect videos Filter UI with the data handling

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to the VideoPress dashboard
* Open the Filter section
* Use the Redux dev tool
* Find the `videos` state
* Confirm the state updates by clicking on the filter checkboxes


https://user-images.githubusercontent.com/77539/195645642-bc5fbcad-bebc-4a75-b4da-ac49f973bec7.mov
